recipes/abpoa/meta.yaml

```diff
@@ -31,3 +31,7 @@ about:
   license: GPL
   license_file: LICENSE
   summary: 'abPOA: fast SIMD-based partial order alignment using adaptive band'
+
+extra:
+  additional-platforms:
+    - linux-aarch64
```

Comment:
Maybe the YAML syntax is not OK here because the CircleCI says that there is no `linux-aarch64` additional platform: https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/112425/workflows/af2aad3a-d875-4947-a770-be6913f52df7/jobs/183259?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary